src/lib/mina_generators/user_command_generators.ml

```diff
@@ -37,7 +37,22 @@ let zkapp_command_with_ledger ?num_keypairs ?max_account_updates
     Option.value num_keypairs
       ~default:((max_account_updates * 2) + (max_token_updates * 3) + 2)
   in
-  let keypairs = List.init num_keypairs ~f:(fun _ -> Keypair.create ()) in
+  let existing_keypairs =
```

Comment:
Since this is not used elsewhere, could you move this under the `keypairs` block of code?